src/data/polynomial/laurent.lean

```diff
@@ -294,6 +307,46 @@ left_inverse_trunc_to_laurent.injective
   f.to_laurent = g.to_laurent ↔ f = g :=
 ⟨λ h, polynomial.to_laurent_injective h, congr_arg _⟩
 
+lemma exists_T_pow (f : R[T;T⁻¹]) :
+  ∃ (n : ℕ) (f' : R[X]), f'.to_laurent = f * T n :=
```

Comment:
I feel like there's a constructive statement with an explicit witness hiding here, in terms of `int_trailing_degree`

Reply:
This is certainly true: `f * T (- f.int_trailing_degree) = to_laurent (something)` and moreover `something`is either `0` or has non-zero constant term.  Should I roll my sleeves up and start defining `laurent_polynomial.<degrees>`?